src/main/kotlin/app/cash/licensee/dependencyGraph.kt

```diff
@@ -229,10 +241,13 @@ internal fun loadPomInfo(
         depth = depth + 1
       )
       licenses += parentPomInfo.licenses
+      if (scm == null && parentPomInfo.scm != null) {
```

Comment:
If you happen to edit this file the second null check isn't really needed. Not a big deal though.